src/sentry/api/endpoints/sentry_internal_app_tokens.py

```diff
@@ -16,6 +16,14 @@
 class SentryInternalAppTokensEndpoint(SentryAppBaseEndpoint):
     permission_classes = (SentryInternalAppTokenPermission, )
 
+    @requires_feature('organizations:sentry-apps', any_org=True)
```

Comment:
Wasn't this feature recently removed?

Reply:
We preemptively created the PR so we're ready for launch. It's still effective today, though.